src/cryptography/hazmat/backends/openssl/x25519.py

```diff
@@ -4,20 +4,69 @@
 
 from __future__ import absolute_import, division, print_function
 
+import warnings
+
 from cryptography import utils
 from cryptography.hazmat.backends.openssl.utils import _evp_pkey_derive
+from cryptography.hazmat.primitives import serialization
 from cryptography.hazmat.primitives.asymmetric.x25519 import (
     X25519PrivateKey, X25519PublicKey
 )
 
 
+_X25519_KEY_SIZE = 32
+_PEM_DER = (
```

Comment:
Can we move this constant to `serialization`, now that we have in at least 2x places?